.travis.yml

```diff
@@ -47,6 +47,7 @@ before_install:
     - conda config --add channels https://conda.binstar.org/cdeepakroy
 
 install:
+    - conda install --yes libgfortran==1.0 numpy==1.10.2 scipy==0.16.0 scikit-image==0.12.3
```

Comment:
Add openslide-python==1.1.0 to this?

I built it for conda and put it in this channel -- [https://conda.anaconda.org/cdeepakroy](https://conda.anaconda.org/cdeepakroy)